src/sidebar.ts

```diff
@@ -97,149 +106,151 @@ const quickstartSidebar = {
-      ],
+      items: flat
+        ? []
+        : [
+            "getting-started/ory-network-oauth2",
+            "getting-started/oauth2-openid/expressjs",
+          ],
     },
     {
       type: "category",
       label: "Permissions",
       collapsed: false,
       collapsible: false,
-      items: [
-        "guides/permissions/overview",
-        "keto/examples/olymp-file-sharing",
-        "keto/guides/rbac",
-        "guides/gitops",
-      ],
+      link: {
+        type: "doc",
+        id: "guides/permissions/overview",
+      },
+      items: flat
+        ? []
+        : [
+            "keto/examples/olymp-file-sharing",
+            "keto/guides/rbac",
+            "guides/gitops",
+          ],
     },
     {
       type: "category",
-      label: "SDK Quickstarts",
+      label: "SDK quickstarts",
       collapsed: false,
       collapsible: false,
-      items: [
-        "kratos/sdk/go",
-        "hydra/sdk/js",
-        "hydra/sdk/go",
-        "keto/sdk/go",
-        "keto/sdk/python",
-      ],
-    },
-  ],
-  troubleshooting: [
-    homeLink,
-    {
-      type: "category",
-      label: "Troubleshooting",
-      collapsible: false,
-      collapsed: false,
       link: {
-        type: "generated-index",
-        slug: "category/troubleshooting/",
+        type: "doc",
+        id: "quickstart/sdks",
       },
-      items: [
-        {
-          type: "autogenerated",
-          dirName: "troubleshooting",
-        },
-        "oauth2-oidc/identity-provider-integration-settings",
-        "guides/upgrade/sdk-v1",
-        "hydra/debug",
-        "hydra/debug/logout",
-        "hydra/debug/token-endpoint-auth-method",
-      ],
+      items: flat
+        ? []
+        : [
+            "kratos/sdk/go",
+            "hydra/sdk/js",
+            "hydra/sdk/go",
+            "keto/sdk/go",
+            "keto/sdk/python",
+          ],
     },
-  ],
-  migrateToOry: [
-    homeLink,
-    "migrate-to-ory/index",
-    "migrate-to-ory/auth0",
-    "hydra/self-hosted/migrating-from-mitreid",
-  ],
+  ]
 }
+const guidesSidebar = (flat: boolean): ExtendSidebar => {
+  const items = []
+  if (!flat) {
+    items.push(homeLink)
+  }
 
-const guidesSidebar = {
-  guides: [
-    homeLink,
-    "actions/live-events",
+  return [
+    ...items,
     {
       type: "category",
-      label: "Authentication methods",
+      label: "Authentication and login",
       collapsed: false,
       collapsible: false,
-      items: [
-        "kratos/concepts/credentials/username-email-password",
-        "kratos/passwordless/one-time-code",
-        "kratos/passwordless/passkeys",
-        "kratos/organizations/organizations",
-        {
-          type: "category",
-          label: "OpenID Connect SSO",
-          items: [
-            "kratos/social-signin/overview",
+      link: flat
+        ? {
+            type: "doc",
+            id: "guides/authentication",
+          }
+        : undefined,
```

Comment:
```suggestion
      link:
         {
            type: "doc",
            id: "guides/authentication",
          },
```

Not quite sure what the undefined part does here? Did you remove the link on purpose? 

Can we link to the category overview doc here in the 2nd level? 

Like we do for Ory Actions for example.

Attached screenshot for clarification - I want every item in the category to be clickable/lead to a doc.
<img width="289" alt="Screenshot 2024-09-23 at 11 50 44" src="https://github.com/user-attachments/assets/fc74ca74-2193-478f-a8e8-b3691bf6e496">

Reply:
Yes, I was experimenting how to get the links working and forgot to clean it up. thank you!. I cleaned it up!